website/docs/r/managed_disk.html.markdown

```diff
@@ -107,7 +107,9 @@ The following arguments are supported:
 
 * `encryption_settings` - (Optional) A `encryption_settings` block as defined below.
 
-* `image_reference_id` - (Optional) ID of an existing platform/marketplace disk image to copy when `create_option` is `FromImage`.
+* `image_reference_id` - (Optional) ID of an existing platform/marketplace disk image or gallery image version to copy when `create_option` is `FromImage`.
+
+* `image_reference_lun` - (Optional) Index of the data disk in the image when copying from a data disk. Defaults to `-1`, which means to copy from os disk.
```

Comment:
this default makes no sense in the context of Terraform - `0` is the default LUN for an OS disk in Azure, which we use elsewhere in the Provider - however as mentioned above, perhaps we shouldn't be exposing this field:

```suggestion
```

Reply:
Have removed the Lun as there is no meaning to expose it here.
While it seems not correct that `0` is the default Lun for an OS disk. e.g. [virtual_machine_data_disk_attachment_resource](https://github.com/hashicorp/terraform-provider-azurerm/blob/82a4cf8727706199cb5c743a8d86f809c86eed33/internal/services/compute/virtual_machine_data_disk_attachment_resource.go#L55-L60) Lun actually starts with 0, and OS disk doesn't occupy the range